### ISSUE TITLE
Sandboxed + Non Sandboxed apps.

### ISSUE THREAD

**Issue by Pinox** [microsoft/winget-cli#386]
please change Windows 10 + package manager so that when we install the app from winget we have an option to sandbox the app or run it natively.  that will give the end user total confidence and will unify the whole ecosystem of sandboxed + non-sandboxed apps. 

It would be the same as you currently run any app on windows 10 with option to "run as admin" or as "normal user"

**Comment by upintheairsheep** [comment]
I agree, to have a system to have sandboxed app containers, preferably one for the program data, one for the data files.

**Comment by denelon** [comment]
@Pinox we've added support for "--scope" to be either "user" or "machine". Not all installers honor both scopes, but we've also added an elevation requirement to the schema as well so packages like Spotify and Postman will inform you when you try to install via administrator that they aren't supported, and other packages that require elevation will do the same. If the installer requires elevation, but it does so itself, you will also be informed that a UAC (User Account Control) prompt is required.

Does that address your ask here, or is there still something else you're looking for?
